public/app/features/alerting/unified/components/receivers/form/GrafanaReceiverForm.tsx

```diff
@@ -114,15 +114,21 @@ export const GrafanaReceiverForm: FC<Props> = ({ existing, alertManagerSourceNam
     ? (existing.grafana_managed_receiver_configs ?? []).some((item) => Boolean(item.provenance))
     : false;
 
-  const readOnly = isVanillaPrometheusAlertManagerDataSource(alertManagerSourceName) || hasProvisionedItems;
+  // this basically checks if we can manage the selected alert manager data source, either because it's a Grafana Managed one
+  // or a Mimir-based AlertManager
+  const isManageableAlertManagerDataSource = !isVanillaPrometheusAlertManagerDataSource(alertManagerSourceName);
+
+  const isEditable = isManageableAlertManagerDataSource && !hasProvisionedItems;
+  const isTestable = isManageableAlertManagerDataSource || hasProvisionedItems;
```

Comment:
It's a bit unclear to me why provisioning impacts the testability of a contact point. Could we add a short explanation?
When exactly we are not able to test a contact point?

Reply:
Sure thing! When a contact-point is provisioned it signals that it should not be editable but still test-able. It doesn't take RBAC in to account though.

We still check if we have an `onTest` function in the actual `<ReceiverForm>`